docs/FAQ.md

```diff
@@ -0,0 +1,75 @@
+# Frequently asked questions
+
+### Why does my branch not move to the new commit after `jj new/commit`?
+
+If your familiar with Git, you might expect the current branch to move forward
+when you commit. However Jujutsu does not have a concept of a "current branch",
+so they always point to a single commit.
```

Comment:
TODO, also change THEM below

Reply:
(Edited the above comment, it got submitted prematurely)

Reply:
Done